a10_octavia/network/drivers/neutron/a10_octavia_neutron.py

```diff
@@ -0,0 +1,709 @@
+#    Copyright 2019, A10 Networks
```

Comment:
I highly doubt this file is needed.
@Hunter, are you gonna use this file in your VLAN solution
@Afrin, do you think you are gonna make any changes in driver in future?

Reply:
I removed it all in my VLAN implementation. This PR was purely to rebase and fast forward, not to refactor and implement new features.